fideslib/oauth/schemas/user_permission.py

```diff
@@ -12,15 +8,6 @@ class UserPermissionsCreate(BaseSchema):
 
     scopes: List[str]
 
-    @validator("scopes")
-    @classmethod
-    def validate_scopes(cls, scopes: List[str]) -> List[str]:
-        """Validates that all incoming scopes are valid"""
-        diff = set(scopes).difference(set(SCOPES))
-        if len(diff) > 0:
-            raise InvalidScopeError(scopes)
-        return scopes
-
```

Comment:
Did this just end up being a FastAPI bug?

Reply:
I tried recreating the issue with just Pydantic and wasn't able to, but I was also never able to figure out where FastAPI was causing the error using the debugger. Maybe it's a combination of both together 🤷

Reply:
hm ok this works for now